### PR TITLE
Adding Darkflow support by generating XML files from .txt file.

### PR DESCRIPTION
**Pull request by jatinmandav**
Script to generate XML Files from .txt file for Darkflow

### CODE REVIEW

**Comment by jatinmandav** [comment]
Yes sure! I just added it so that if the user has a huge amount to data to convert the user would  now how much is completed. I will modify the file. :)

File: generate_xml.py [hunk 1]
```diff
@@ -91,42 +90,39 @@ def yolo_to_x_y(x_center, y_center, x_width, y_height, width, height):
 for i in range(len(labels)):
     labels[i] = labels[i].replace('\n', '')
```
Comment (Cartucho) [inline generate_xml.py]: Could you please add here:
```
if not os.path.exists(saveDir):
    os.makedirs(saveDir)
```

**Comment by jatinmandav** [comment]
@Cartucho I have updated the file. :)

**Comment by Cartucho** [comment]
Thank you! :1st_place_medal:

**Comment by jatinmandav** [comment]
Hi
Thanks @lucascimeca for pointing out. I have fixed it.

**Pull request merged**